src/jobflow_remote/cli/job.py

```diff
@@ -1,6 +1,8 @@
+from __future__ import annotations
```

Comment:
This breaks the CLI. I always get an error 
> RuntimeError: Type not yet supported: <class 'jobflow_remote.cli.types.DictType'>

that goes away if remove this line and get back to `Optional` instead of `None`

Reply:
Here are the related issues: https://github.com/tiangolo/typer/issues/533 and https://github.com/tiangolo/typer/issues/348 -- for now we can disable this kind of rewrite in pyupgrade (I will try to go through and check that there are no other occurences)